CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Fixed
+* **dso-toolkit + styling:** Definition-list (in accordion) geen padding bij lange dt ([#979](https://github.com/dso-toolkit/dso-toolkit/issues/979))
```

Comment:
Er zitten geen changes in package "styling", right? alleen in toolkit!

Reply:
Correct. Gedachtenkronkel van mij.